1.1 Desktop app calls Web API - PoP/TodoListClient/TokenCacheHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Security.Cryptography;
+using Microsoft.Identity.Client;
+
+namespace TodoListClient
+{
+    static class TokenCacheHelper
```

Comment:
You should add a comment that this helper is ok for Windows only and point to the extension for Linux and Mac.

Reply:
(I know POP is only available on .net classic for now..)

Reply:
Yes. BTW @TiagoBrenck has created a cross platform sample for desktop; https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/pull/4

Reply:
Thanks @bgavrilMS 
Done